weasyl/character.py

```diff
@@ -359,10 +406,16 @@ def remove(userid, charid):
     return ownerid
 
 
-def fake_media_items(charid, userid, login, settings):
-    submission_url = define.cdnify_url(
-        define.url_make(charid, "char/submit", query=[userid, settings], file_prefix=login))
-    cover_url = define.cdnify_url(define.url_make(charid, "char/cover", query=[settings], file_prefix=login))
+def fake_media_items(charid, userid, login, settings, absolutify=False):
+    submission_url = define.cdnify_url(define.url_make(charid, "char/submit", file_prefix=login))
```

Comment:
This now ignores its setting parameter, which may break other users. Checking 'Find usages' in pycharm finds select_list() for example.

We may have to spend some time thinking about how to do this right. We could do d.engine.execute() instead of using the nice orm interface and return something more like a raw dict.

We could make define.url_make() take settings in the style of the orm, but that's going to be a lot of code changed in this file too.

We could also do the ugly brittle thing of knowing that `define.url_make` only cares about `file_type` and reconstruct a 'settings' in the old style by reversing the file_type map: https://github.com/Weasyl/libweasyl/blob/master/libweasyl/models/helpers.py#L89
This would work but breaks abstraction barriers.

If this looks like deciding the right thing to do is taking too long, let's separate out the journal endpoint change so it's not blocked on getting this right.